src/OpenSSL/SSL.py

```diff
@@ -2369,7 +2414,7 @@ def bio_shutdown(self) -> None:
 
         _lib.BIO_set_mem_eof_return(self._into_ssl, 0)
 
-    def shutdown(self) -> bool:
+    def shutdown(self) -> bool:  # type: ignore[return]
```

Comment:
We ignore the return type, because `mypy` is unable to know that `_raise_ssl_error` is sure to bomb.